codacy/values-production.yaml

```diff
@@ -196,9 +196,12 @@ listener:
     host: <--- codacy-db-host --->
     service:
       port: 5432
+  # These configurations are only used if the `nfsserverprovisioner` is enabled
   persistence:
     claim:
-      size: 140Gi
+      # This is the size of the claim that the component will get from the provisioner (if used)
+      # Therefore this value needs to be smaller than `nfsserverprovisioner.persistence.size`
+      size: 190Gi
```

Comment:
shouldn't we do it like this? is it even possible? 😅 

`size: {{ sub Values.nfsserverprovisioner.persistence.size 10 }}`

Reply:
I see the reasoning, having a margin of 10 Gb isn't excessive?